pkg/reconciler/contour/config/contour.go

```diff
@@ -54,13 +58,22 @@ type visibilityValue struct {
 // NewContourFromConfigMap creates an Contour config from the supplied ConfigMap
 func NewContourFromConfigMap(configMap *corev1.ConfigMap) (*Contour, error) {
 	var tlsSecret *types.NamespacedName
+	var timeoutPolicyResponse string
+	var timeoutPolicyIdle string
```

Comment:
These should be defaulted to 'infinity' here

The properties added to `config-contour.yaml` are actually in an `_example` block. We don't set properties explicitly there since we don't want to override user settings when upgrading

Reply:
Done